src/server/.env

```diff
@@ -1,3 +1,3 @@
-ATLAS_URI=mongodb+srv://tn9nex:Ddrm1573559@relative-studios-wzmvi.mongodb.net/react-todo
```

Comment:
Don't push the .env file up when creating a pull request, I'll delete it for you.